lib/src/migration_visitor.dart

```diff
@@ -113,6 +113,11 @@ abstract class MigrationVisitor extends RecursiveAstVisitor {
   void visitDependency(Uri dependency, FileSpan context) {
     var result = importCache.import(dependency, _importer, _currentUrl);
     if (result != null) {
+      // If [dependency] comes from a non-relative import, don't migrate it,
+      // because it's likely to be outside the user's repository and may even be
+      // authored by a different person.
+      if (result.item1 != _importer) return;
```

Comment:
This definitely makes sense for the default behavior, but would it make sense to add a flag to override this?

I think there are some stylesheets within Google that specify all dependencies relative to a load path.

Reply:
Added a TODO.